index.src.html

```diff
@@ -378,7 +378,7 @@
       Promise&lt;Credential?&gt; get(optional CredentialRequestOptions options);
       Promise&lt;Credential&gt; store(Credential credential);
       Promise&lt;Credential?&gt; create(optional CredentialCreationOptions options);
-      Promise&lt;void&gt; requireUserMediation();
+      Promise&lt;void&gt; preventSilentAccess();
```

Comment:
I'm not thrilled with this name, but I haven't thought of a better one since we discussed it. LGTM.